### PR TITLE
fast commit

### PR DESCRIPTION
**Pull request by citee520**

### CODE REVIEW

File: rarbgsubscriber/daemon.py [hunk 1]
```diff
@@ -144,7 +144,7 @@ def start(self, *args, **kwargs):
             sys.exit(1)
 
         # Start the daemon
-        self.daemonize()
```
Comment (verybada) [inline rarbgsubscriber/daemon.py:147]: Why remove daemonize, is it for debugging?

File: rarbgsubscriber/rarbg.py [hunk 2]
```diff
@@ -26,14 +26,7 @@
...
         # TODO: how to pass bot check
-        self._cookie = """
-            c_cookie=ljd3gcszby;
-            expla2=1%7CSat%2C%2005%20Mar%202016%2021%3A46%3A26%20GMT;
-            LastVisit=1457192974; vDVPaqSe=r9jSB2Wk; expla=2; tcc;
-            __utma=9515318.355702320.1444490883.1457101385.1457192786.76;
-            __utmb=9515318.4.10.1457192786; __utmc=9515318;
-            __utmz=9515318.1447162163.12.4.utmcsr=
-            google|utmccn=(organic)|utmcmd=organic|utmctr=(not%20provided)"""
+        self._cookie = 'c_cookie=ljd3gcszby; expla2=1%7CSat%2C%2005%20Mar%202016%2021%3A46%3A26%20GMT; LastVisit=1457192974; vDVPaqSe=r9jSB2Wk; expla=2; tcc; __utma=9515318.355702320.1444490883.1457101385.1457192786.76; __utmb=9515318.4.10.1457192786; __utmc=9515318; __utmz=9515318.1447162163.12.4.utmcsr=google|utmccn=(organic)|utmcmd=organic|utmctr=(not%20provided)'
```
Comment (verybada) [inline rarbgsubscriber/rarbg.py]: against pep8, max chararcters per line

File: rarbgsubscriber/rarbg.py [hunk 3]
```diff
@@ -73,6 +66,13 @@ def _parse(self, tr):
         result['href'] = urlparse.urljoin(self._host, info_tag.a['href'])
         result['imdb'] = self._get_imdb(imdb_tag)
         result['size'] = size_tag.text
+		
+        imaged_praser = info_tag.a['onmouseover']
```
Comment (verybada) [inline rarbgsubscriber/rarbg.py]: typo "praser"

File: rarbgsubscriber/rarbg.py [hunk 4]
```diff
@@ -73,6 +66,13 @@ def _parse(self, tr):
...
+        image_jpg_url = re.match('.+src=\\\\\'(.+)\\\\\' ', imaged_praser)
+        image_url= urlparse.urljoin('http:', image_jpg_url.group(1))
+        
+        result['image'] = image_url
+        LOG.info("result['image']==%s " % result['image'])
```
Comment (verybada) [inline rarbgsubscriber/rarbg.py:73]: for debugging?

File: temp_conf.json [hunk 5]
```diff
@@ -1,11 +1,11 @@
 {
```
Comment (verybada) [inline temp_conf.json:1]: do not add any personal information, this is a template

File: rarbgsubscriber/rarbg.py (hunk 3 above)
Comment (citee520) [inline rarbgsubscriber/rarbg.py]: already fix it to parser

File: rarbgsubscriber/daemon.py (hunk 1 above)
Comment (citee520) [inline rarbgsubscriber/daemon.py:147]: i want print message,so close it first

File: temp_conf.json (hunk 5 above)
Comment (citee520) [inline temp_conf.json:1]: ok, i will remove this file.

File: rarbgsubscriber/rarbg.py (hunk 4 above)
Comment (citee520) [inline rarbgsubscriber/rarbg.py:73]: yes, i want verify it can get corrent info.if you don't need, i will remove it.

**Pull request closed without merging**